File: CWPopupDemo/SamplePopupViewController.m
//
//  SamplePopupViewController.m
//  CWPopupDemo
//
//  Created by Cezary Wojcik on 8/21/13.
//  Copyright (c) 2013 Cezary Wojcik. All rights reserved.
//

#import "SamplePopupViewController.h"

@interface SamplePopupViewController ()

@end

@implementation SamplePopupViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    // use toolbar as background because its pretty in iOS7
    UIToolbar *toolbarBackground = [[UIToolbar alloc] initWithFrame:CGRectMake(0, 44, 200, 106)];
    [self.view addSubview:toolbarBackground];
    [self.view sendSubviewToBack:toolbarBackground];
    // set size
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: CWPopup-Swift/CWPopup-Swift-Demo/CWPopup-Swift-Demo/SamplePopupViewController.swift
//
//  SamplePopupViewController.swift
//  DimPresentViewController
//
//  Created by admin on 3/8/16.
//  Copyright © 2016 __ASIAINFO__. All rights reserved.
//

import UIKit

class SamplePopupViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        let toolBarBG = UIToolbar(frame: CGRect(x: 0, y: 44, width: 200, height: 106))
        view.addSubview(toolBarBG)
        view.sendSubviewToBack(toolBarBG)
    }

}
